app/views/addresses/edit.html.slim

```diff
@@ -1,3 +1,3 @@
 .Address-form-container
-  h1 Adresse bearbeiten
+  h1= t('address.actions.edit')
```

Comment:
Hi @gmichokostas, would you please reference the locales in the views according to "lazy loading"? In this case you would need to use t('.heading') and in the YAML file just put the string under 'de.addresses.edit.heading'. And you would rename the YAML file to "views/addresses/de.yml".
The translations for the welcome#index view are already in this lazy loading format.
Thanks!